src/stream/tests/integration_tests/snapshot.rs

```diff
@@ -24,6 +28,23 @@ use risingwave_stream::executor::{BoxedMessageStream, Message};
 pub struct SnapshotOptions {
     /// Whether to sort the output chunk, required if the output chunk has no specified order.
     pub sort_chunk: bool,
+
+    /// Whether to include the result after applying the changes from each output chunk. One can
+    /// imagine this as the result of a `SELECT * FROM mv` after each output chunk.
+    pub include_applied: bool,
```

Comment:
`applied` doesn't seem to easy to understand. It's more like "snapshot" (but this would be more confusing), or "aggregated" (but this is also confusing) 🤣

Reply:
"accumulated"? 🤔

Reply:
Yeah I once named it "aggregated". 🤣

Reply:
What about `include_full_result`?

Reply:
I renamed it to "include_applied_result". Does it LGTY? 🥵